apps/src/code-studio/pd/workshop_dashboard/workshop.jsx

```diff
@@ -104,6 +104,14 @@ const Workshop = React.createClass({
           'account_required_for_attendance?'
         ])
       });
+    }).fail(data => {
+      if (data.statusText !== "abort") {
+        console.log(`Failed to get workshop: ${this.props.params.workshopId}`);
```

Comment:
Sort of initially (I started by copying one of the other places where we display an alert and log to the console), but I like just displaying the "No workshop found" message and not logging anything so I'll remove it. Thx